docs/Training-SAC.md

```diff
@@ -223,11 +223,11 @@ Typical Range: `4` - `128`
 ### Memory Size
 
 `memory_size` corresponds to the size of the array of floating point numbers
-used to store the hidden state of the recurrent neural network. This value must
-be a multiple of 4, and should scale with the amount of information you expect
+used to store the hidden state of the recurrent neural network in the policy.
+This value must be divisible by 2, and should scale with the amount of information you expect
```

Comment:
`This value must be a multiple of 2` for consistency with PPO

Reply:
Changed - also changed LearningModel to ModelUtils and made the optimizer methods (and SACNetwork methods) private.